### PR TITLE
Bug fixing and implementing new functionalities based on requests

### PR DESCRIPTION
**Pull request by Ducksss**
# Bug fixing and implementing new functionalities based on requests
Following Yu Fang's discovery of some bugs and suggestions, such as the date not being parsed on iOS devices (iOS only enables parsing of date based on ISO8601 Standard), this PR was created to address this issue. It should also include changes to redirect the user to newsletter/developer blogs rather than expanding for more information based on Darren's recommendation. Thus, I've also updated the test cases to suit the changes.

image

# Changelog
- Fixing broken links such as the link to subscribe to newsletter. (_includes/site-section-tabs.html)
- Limiting rendering of cards to just 3 instead of visually hiding them in the DOM (_includes/site-section-tabs.html)
- Implemented Developer Blog content (_includes/site-section-tabs.html)
- Minor CSS Update for Calendar Icon on mobile devices (apps/src/search/SearchAllEvents.vue)
- Code formatting (apps/src/communities/conferences-webinars-sort.js)
- Deferring of script until content loads (collections/_communities/overview.html)
- Added new settings such as chromewebsecurity to allow testing of diff pages other than developer.gov.sg pages
- Updating urls to be more dynamic such as relying based on basepath instead of static url (cypress/integration/community-section-tab/all-events.spec.js)
- Allowing the usage on {force:click} on anchor tags cypress/support/index.js
- Updating test files and accounting for edge cases in the future (rest of **.spec.js)

### CODE REVIEW

**Comment by Ducksss** [comment]
@yanbutan lemme check the application on iOS devices first before you approve 😗

**Comment by Ducksss** [comment]
Alright, nice, it's displaying properly in iOS devices now hahaha

image

So is the developer blogs section :)
image

File: apps/src/search/SearchAllEvents.vue [hunk 1]
```diff
@@ -195,8 +207,11 @@ export default {
       document.getElementById("query-all-year").value = dateParam;
       document.getElementById("query-all-category").value = categoryParam;
 
-      for (var i = 0; i < this.searchResults.length; i++) {
-        const dt = new Date(this.searchResults[i].event_date_raw);
+      for (let i = 0; i < this.searchResults.length; i++) {
+        const t = this.searchResults[i].event_date_raw.split(/[- :]/);
+        const d = new Date(t[0], t[1] - 1, t[2], t[3], t[4], t[5]);
+        const dt = new Date(d);
+
```
Comment (yanbutan) [inline apps/src/search/SearchAllEvents.vue:214]: I think maybe add some comments on why formatting was done this way.

**Comment by yanbutan** [comment]
Looks Good bruh, dint really check the cypress code but the other codes look fine.

**Review by yanbutan — APPROVED** [review]

File: apps/src/search/SearchAllEvents.vue (hunk 1 above)
Comment (Ducksss) [reply apps/src/search/SearchAllEvents.vue:216]: doneeeee

**Comment by Ducksss** [comment]
> Looks Good bruh, dint really check the cypress code but the other codes look fine.

thanks for looking at the code bro 🤝 😸
